test/src/test/java/org/corfudb/integration/ServerRestartTest.java

```diff
@@ -132,6 +132,37 @@ public void testSequencerRecoveryFailoverServerAndClient() throws Exception {
     }
 
     /**
+     * Test recovery with a failed over server and a new client.
+     *
+     * @throws Exception
+     */
+    @Test
+    public void testDoubleFailoverRecovery() throws Exception {
```

Comment:
Isn't this test exactly same as the one above this? testSequencerFailoverServerAndClient.
Secondly, contrary to the name there is only one failover taking place in the test.